aws/resource_aws_rds_cluster.go

```diff
@@ -758,13 +758,12 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 		}
 
 		createOpts := &rds.CreateDBClusterInput{
-			CopyTagsToSnapshot:   aws.Bool(d.Get("copy_tags_to_snapshot").(bool)),
-			DBClusterIdentifier:  aws.String(identifier),
-			DeletionProtection:   aws.Bool(d.Get("deletion_protection").(bool)),
-			Engine:               aws.String(d.Get("engine").(string)),
-			EngineMode:           aws.String(d.Get("engine_mode").(string)),
-			ScalingConfiguration: expandRdsScalingConfiguration(d.Get("scaling_configuration").([]interface{})),
```

Comment:
This change is valid because the attribute is handled later (line 853), however we may want to do the opposite removal here for consistency with the other `CreateDBClusterInput` handling in this file which allow `expandRdsScalingConfiguration()` do the right thing with the missing attribute.